src/_includes/assets/css/screen.css

```
@@ -2389,10 +2400,13 @@ a.banner:hover span {
   }
 
   .ad-layout .sidebar {
+    background-color: var(--gray05);
     display: flex;
-    max-width: 300px;
+    width: 320px;
     flex-direction: column;
     justify-content: space-between;
+    align-items: center;
+    padding: 0 10px 10px 10px;
```

This should get the CodeFactor test to pass:

```suggestion
    padding: 0 10px 10px;
```